.github/ISSUE_TEMPLATE/aws-access-request.yml

```diff
@@ -0,0 +1,90 @@
+
+name: WIP AWS Access Request
+description: To request access to AWS
+title: Access for [individual]
+labels: ['external-request', 'operations', 'ops-access-request']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        1. Change the **Title** to include your name
+        2. Add the label used by your team (eg: `BAH-526`)
+        3. Do not remove `operations` label or the `ops-access-request` label; this will notify the VSP Operations team of your request
+        4. Add additional info as a comment after GitHub issue is created if you select 'Other' for any of the form inputs.
+        5. When the issue is closed you will be notified and can continue with on-boarding setup
+  - type: dropdown
+    id: cor-name
+    attributes:
+      label: COR Name
+      description: Your Contracting Officer's Representative (COR) name.
+      options:
+        - Cecila Lee
+        - Courtney Bethea
+        - Crystal Moultrie
+        - Doris Lin
+        - Vilay Senthep
```

Comment:
I replaced Mary with Vilay

Reply:
I was provided this information today, "Vilay is not an acting COR and if the approval is sent to her for any part of the process she will not receive it". Take this with a grain of salt, bc I'm not familiar with _any_ of these individuals. I do know that my COR is Eunice Garcia, if you want to add them.

Reply:
Thanks for this info, Kyle! I'm not familiar with any of them either 😂 . Adding Eunice.